librz/analysis/meson.build

```diff
@@ -165,6 +148,31 @@ rz_analysis_sources = [
   '../asm/arch/luac/v53/opcode_53.c'
 ]
 
+if capstone_dep.version().split('.')[0].to_int() > 4
+  rz_analysis_sources += 'p/analysis_riscv_cs.c'
+endif
+
+if get_option('use_gpl')
+  rz_analysis_sources += [
+    'p/analysis_arc.c',
+    'p/analysis_arm_gnu.c',
+    'p/analysis_cris.c',
+    'p/analysis_mips_gnu.c',
+    'p/analysis_nios2.c',
+    'p/analysis_ppc_gnu.c',
```

Comment:
Just remove PPC GNU completely, it seems also unmaintained and untested for years.

Reply:
Could be done in a separate PR, like I did for ARM GNU and WineDbg